go/obscuronode/enclave/evm/evm_facade.go

```diff
@@ -33,9 +33,9 @@ func ExecuteTransactions(txs []*nodecommon.L2Tx, s *state.StateDB, header *nodec
 		}
 		receipts = append(receipts, r)
 		if r.Status == types.ReceiptStatusFailed {
-			log.Info("Unsuccessful (status != 1) tx %d.", obscurocommon.ShortHash(t.Hash()))
+			log.Info("Unsuccessful (status != 1) tx %d. Logs: %+v", obscurocommon.ShortHash(t.Hash()), r.Logs)
```

Comment:
Is this worth doing? When I run the full sim locally, the receipts are always empty.

Reply:
removing